src/Log/EventService.php

```diff
@@ -39,9 +40,10 @@ public function __construct(EntityManagerInterface $em, TokenStorageInterface $t
         $this->refl = $refl;
     }
 
-    public function log(AbstractEvent $event)
+    public function log(AbstractEvent $event): void
     {
         $entity = $this->hydrate($event);
+        assert($entity !== null);
```

Comment:
This assert now makes sense because (generally) `hydrate()` only returns null if it was given null as an argument and `$event` is defined as non nullable. However, that obviousness might change with future code. For that reason, please add a comment indicating the reasoning behind this assert.